pyglossary/glossary.py

```diff
@@ -1236,6 +1236,33 @@ def writeDict(self, filename: str = "", writeInfo: bool = False) -> None:
 			ext=".dict",
 		)
 
+	def	iterJsonLines(
+		self,
+		filename: str = "",
+		infoKeys: Optional[List] = None,
+		addExtraInfo: bool = True,
+		newline: str = "\\n",
+		transaction: bool = False,
+	) -> Iterator[str]:
+		newline = "<br>"
+
+		yield (
+			"{"
+		)
+
+		for i, entry in enumerate(self):
+			if entry.isData():
+				# FIXME
+				continue
+			word = entry.word.replace("\\", "\\\\").replace('"', '\\"').replace("\n","").replace("\r","")
```

Comment:
Use `json.dumps(word)` and `json.dumps(defi)`

Reply:
Tried json.dumps() with one of my JSONs, didn't work due to some reasons (produced file was not valid), reverted to string.replace()....